### ISSUE TITLE
Can't route to "5975 Park Avenue, Richmond, 94805"

### ISSUE THREAD

**Issue by graue** [bikehopper/graphhopper#87]
Consistently getting this error for routes to this location (e.g. from "Mission District")

java.lang.IllegalStateException: Speed cannot be 0 for unblocked edge, use access properties to mark edge blocked! Should only occur for shortest path calculation. See #242.
        at com.graphhopper.routing.weighting.AbstractWeighting.calcEdgeMillis(AbstractWeighting.java:78)
        at com.graphhopper.routing.weighting.FastestWeighting.calcEdgeMillis(FastestWeighting.java:113)
        at com.graphhopper.gtfs.GraphExplorer.walkPath(GraphExplorer.java:264)
        at com.graphhopper.gtfs.TripFromLabel.transferPath(TripFromLabel.java:435)
        at com.graphhopper.gtfs.TripFromLabel.parsePartitionToLegs(TripFromLabel.java:384)
        at com.graphhopper.gtfs.TripFromLabel.createResponsePath(TripFromLabel.java:82)
        at com.graphhopper.gtfs.PtRouterImpl$RequestHandler.parseSolutionsAndAddToResponse(PtRouterImpl.java:190)
        at com.graphhopper.gtfs.PtRouterImpl$RequestHandler.route(PtRouterImpl.java:183)
        at com.graphhopper.gtfs.PtRouterImpl.route(PtRouterImpl.java:77)
        at com.graphhopper.resources.PtRouteResource.route(PtRouteResource.java:92)
        at jdk.internal.reflect.GeneratedMethodAccessor26.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167)
        at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:475)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:397)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:81)
        at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:255)
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
        at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)
        at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:234)
        at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680)
        at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394)

**Comment by graue** [comment]
Duplicate of #68
